src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -76,9 +88,15 @@ const ProfilePictureInput: React.FunctionComponent<PropsType> = ({
   };
 
   const uploadDefaultAvatar = async (avatar: string) => {
-    setValue("pictureUrl", process.env.PUBLIC_URL + `/avatars/${avatar}`, true);
+    setValue("pictureUrl", avatar, true);
   };
```

Comment:
Why getting rid of `process.env.PUBLIC_URL` here? Presumably that would still be relevant to include in the `sparkleAvatars` array above?

Reply:
The default avatars are in the public/assets location and there is no need for this `process.env.PUBLIC_URL` The avatars are still included even without this, I double checked.

Reply:
> and there is no need for this `process.env.PUBLIC_URL`

That's only because we currently build the app to be hosted at `/`, so it works as a fluke, not because it's correct. Using `process.env.PUBLIC_URL` is the correct way for this to be done:

- https://create-react-app.dev/docs/using-the-public-folder/#adding-assets-outside-of-the-module-system